### DISCUSSION TITLE
Adding scripts to start and stop Docker containers with actual ports used in Testing

### DISCUSSION THREAD

**Discussion by rokerzfirst101** [gofr-dev/gofr#421] [Ideas]
Going into each example/test and finding the ports to run the containers on might be exhausting for new contributors. Instead, a script could be added to the Contributing guide which runs the docker containers on the actual test ports.

Similarly, we could also look into a script to stop all GoFr related containers.

This would allow users quick access to starting and stopping the containers when required.

**Comment by vikash** [comment]
We intend to have a README in each of the projects where the docker command is written. IMO, having a one liner docker command is easier to understand, copy and execute than having a script containing that one line and executing that script. Most of the examples are intended to showcase only one concept and thus the docker execution itself is very simple. 

Having a common script to run all gofr related containers may be an overkill for majority of folks as we intend to add more and more different databases and container examples in future. So, people will need to run 10s of containers even when they just want to execute an example with redis.

**Reply by rokerzfirst101** [reply]
The use case I had in mind was for folks wanting to run all the containers required for testing locally. 

Also, scripts aren't looking like the right choice now. I'll try switching to a Docker Compose file for personal use.
